<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>975</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>900</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>700</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Adobe 楷体 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>矩形平面</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Adobe 楷体 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>圆柱面</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>430</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Adobe 楷体 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>扇形面</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>211</width>
           <height>118</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="2">
           <widget class="QLineEdit" name="plane_end2y">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>起点1：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>起点2：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="plane_start2x">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="plane_start2y">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="plane_start2z">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>终点2：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="plane_end2x">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="plane_end2z">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="plane_start1x">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="plane_end1z">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="plane_end1x">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="plane_start1z">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="plane_end1y">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="plane_start1y">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>终点1：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>380</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>折弯角度:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ringAngle">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>580</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>-90</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>250</y>
           <width>231</width>
           <height>118</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>轴线起点：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="generatrix_endx">
            <property name="text">
             <string>300</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="generatrix_startz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>母线终点：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="generatrix_starty">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="generatrix_endy">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="generatrix_endz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="axis_startx">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="axis_starty">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="axis_startz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="axis_endx">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="axis_endy">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="axis_endz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>母线起点：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="generatrix_startx">
            <property name="text">
             <string>300</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>轴线终点：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_16">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_17">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_18">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>460</y>
           <width>231</width>
           <height>118</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="0">
           <widget class="QLabel" name="label_19">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>轴线起点：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="ringGeneratrix_endx">
            <property name="text">
             <string>300</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="ringGeneratrix_startz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_20">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>母线终点：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="ringGeneratrix_starty">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="ringGeneratrix_endy">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="ringGeneratrix_endz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="ringAxis_startx">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="ringAxis_starty">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="ringAxis_startz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="ringAxis_endx">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="ringAxis_endy">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="ringAxis_endz">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_21">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>母线起点：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ringGeneratrix_startx">
            <property name="text">
             <string>300</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_22">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>轴线终点：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_23">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_24">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_25">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>   Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_26">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>580</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>折弯角度:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_27">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>610</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>板料厚度：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="thickness">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>610</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>50</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>652</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>平面</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>650</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>圆柱面</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>650</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>扇形面</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cylinderAngle">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>380</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>-90</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
